projects/fal/src/fal/cli/auth.py

```diff
@@ -45,23 +44,42 @@ def _list_accounts(args):
     config = Config()
     current_account_name = config.get_internal("team") or user_access.info["nickname"]
 
-    table = Table(border_style=Style(frame=False), show_header=False)
-    table.add_column("#")
-    table.add_column("Nickname")
-    table.add_column("Type")
-
-    for idx, account in enumerate(user_access.accounts):
-        selected = account["nickname"] == current_account_name
-        color = "bold yellow" if selected else None
-
-        table.add_row(
-            f"* {idx + 1}" if selected else f"  {idx + 1}",
-            account["nickname"],
-            "Personal" if account["is_personal"] else "Team",
-            style=color,
-        )
+    if args.output == "json":
+        json_accounts = []
+        for idx, account in enumerate(user_access.accounts):
+            selected = account["nickname"] == current_account_name
+            json_accounts.append(
+                {
+                    "index": idx + 1,
+                    "nickname": account["nickname"],
+                    "type": "Personal" if account["is_personal"] else "Team",
```

Comment:
Unsure here but don't we usually use lowercase types / states when machine readable?

Reply:
adjusted, thanks!